src/components/page-tabs/page-tabs.styles.ts

```diff
@@ -7,7 +7,7 @@ import { getMediaQueryMargins } from '@/utils/media-query/get-media-queries-marg
 export const overrides = {
   tabs: {
     Root: {
-      style: ({ $theme }: { $theme: Theme }): StyleObject => ({
+      style: (x: any, __z: any): StyleObject => ({
```

Comment:
```suggestion
      style: (): StyleObject => ({
```

Or you could make "style" a plain object instead of a function